### ISSUE TITLE
Add new Resources

### ISSUE THREAD

**Issue by ayush-Shandilya** [ayush-Shandilya/STILLLEARN#2]

**Comment by kamallochan2004** [comment]
@ayush-Shandilya 
Can you specify what kind of resources are required?

**Comment by ayush-Shandilya** [comment]
@kamallochan2004 In the relevant Resource card, provide an introduction, getting started, and some free language resources for the language.

**Comment by kamallochan2004** [comment]
@ayush-Shandilya Please check the PR I have added relevant Resource card, to provide an introduction, getting started.

**Comment by Priyanshu-su30** [comment]
Please assign me this task.

**Comment by charu1603** [comment]
I want to contribute to this issue, please assign me

**Comment by ayush-Shandilya** [comment]
Yeah, Simply submit a pull request. After evaluating it, I will merge it.
